### PR TITLE
make the charpage optional for `std clip`

### PR DESCRIPTION
**Pull request by amtoine**
related to
- https://discord.com/channels/601130461678272522/615253963645911060/1142060647358668841

# Description
in order to make the charpage for Windows as general as possible, `chcp` will only run on Windows when `--charpage` is given an integer.

while i was at it, i fixed the system messages given to `check-clipboard` because some of the were incorrect => see the second commit 6865ec9a5

# User-Facing Changes
this is a breaking change as users relying on the fact that `std clip` changed the page to `65001` by itself is not true anymore => they will have to add `--charpage 65001`.

# Tests + Formatting

# After Submitting

### CODE REVIEW

File: crates/nu-std/std/mod.nu
Comment (fdncred) [inline crates/nu-std/std/mod.nu:145]: It's called a `codepage` on Windows.

**Review by fdncred — APPROVED** [review]
Other than changing `codepage` vs `charpage` I think this is fine.

File: crates/nu-std/std/mod.nu
Comment (fdncred) [inline crates/nu-std/std/mod.nu:145]:
```suggestion
    --codepage (-c): int  # the id of the codepage to use (only on Windows), see https://en.wikipedia.org/wiki/Windows_code_page, e.g. 65001 is for UTF-8
```

File: crates/nu-std/std/mod.nu
Comment (fdncred) [inline crates/nu-std/std/mod.nu:165]:
```suggestion
            if $codepage != null {
```

File: crates/nu-std/std/mod.nu
Comment (fdncred) [inline crates/nu-std/std/mod.nu:166]:
```suggestion
                chcp $codepage
```

File: crates/nu-std/std/mod.nu
Comment (amtoine) [reply crates/nu-std/std/mod.nu]: woopsie :eyes: 

should be better in f6db72a47 :wink:

**Review by fdncred — APPROVED** [review]
looks better now

**Comment by amtoine** [comment]
if we do not need to wait for after the release, then we can land it :ok_hand:

**Pull request merged**
